.travis.yml

```diff
@@ -0,0 +1,21 @@
+## -*- coding: utf-8 -*-
```

Comment:
Any ideas why it's adding this file? It already exists in master : https://github.com/Veritomyx/PeakInvestigator-Python-SDK/blob/master/.travis.yml